docs/_quickstart/invite_team_member.md

```diff
@@ -0,0 +1,65 @@
+| View Integration Versions                                         	| ✓         	| ✓                	| X                       	|
+| Manage Integration Versions                                       	| ✓         	| X                	| X                       	|
+| View and Comment on Bug and Feature Requests                      	| ✓         	| ✓                	| X                       	|
+| Invite Users to Use Integration                                   	| ✓         	| ✓                	| X                       	|
+| View Team Members                                                 	| ✓         	| ✓                	| X                       	|
+| Add Admins as Team Members                                        	| ✓         	| X                	| X                       	|
+| Add Collaborators as Team Members                                 	| ✓         	| ✓                	| X                       	|
+| Remove Team Members                                               	| ✓         	| X                	| X                       	|
+| View Analytics                                                    	| ✓         	| ✓                	| X                       	|
+| Monitor Events and Logs                                           	| ✓         	| X                	| X                       	|
+| View Historical Audit Logs                                        	| ✓         	| ✓                	| X                       	|
+| Zapier Command Line Interface (CLI)                               	| ✓         	| X                	| X                       	|
+| Embed Zapier Partner Tools                                        	| ✓         	| ✓                	| X                       	|
+| Receive Integration Issue Notification Email                      	| ✓         	| X                	| X                       	|
+| Receive Monthly Partner Newsletter Email                          	| ✓         	| ✓                	| ✓                       	|
+
+## Manage Team - Settings
+
+### Allow team members with specific email domains to join your integration as a Collaborator
+
+This setting allows anyone with an email address associated with an approved email domain to join your integration as a Collaborator on their own, without waiting for an admin invitation. This makes it easy for your team members to join and view but not make changes to the integration. When a team member joins your integration this way, all the admins will receive an email notification.
+
+If you entered a Homepage URL in the Integration Settings page, that URL is used as an approved email domain. You can then edit, delete, or add more approved email domains. If there are no approved email domains, your team members cannot join your integration on their own, even if this setting has been enabled.
+
+![Screenshot](https://cdn.zappy.app/d6e565c21b58a7731a5b0b4311ceb201.png)
```

Comment:
Is there a service somewhere where we can upload images and GIFs, rather than relying on Zappy?

Reply:
We could upload them to this repo and serve them via Github... But I frequently see Zappy still being used, and it may be better to keep external links repository-agnostic for when we move over to GitLab